src/main/java/com/sparrowwallet/drongo/wallet/WalletModel.java

```diff
@@ -70,6 +70,19 @@ public boolean isCard() {
         return (this == TAPSIGNER || this == SATSCARD);
     }
 
+    // for card devices that require a PIN code, returns the minimum size of valid PIN code
+    public int getMinPinLength() {
```

Comment:
you can also use this can of switch case statement :
`return switch(this) {
            case TAPSIGNER, SATSCARD -> 6;
            case SATOCHIP -> 4;
            default -> 0;
        };`